src/ontology/ro-edit.owl

```diff
@@ -6395,6 +6396,17 @@ AnnotationAssertion(rdfs:comment obo:RO_0013009 "Not restricting range to 'cell'
 AnnotationAssertion(rdfs:label obo:RO_0013009 "sends synaptic output to cell"@en)
 SubObjectPropertyOf(obo:RO_0013009 obo:RO_0002120)
 
+# Object Property: obo:RO_0013010 (synapses restricted to region)
+
+AnnotationAssertion(obo:IAO_0000112 obo:RO_0013010 "An optic lobe intrinsic neuron has its synapses restricted to optic lobe neuropil.")
+AnnotationAssertion(obo:IAO_0000115 obo:RO_0013010 "A relationship between a neuron and a region, where all of the functionally relevant input and output synapses are in that region.")
+AnnotationAssertion(terms:creator obo:RO_0013010 <https://orcid.org/0000-0002-1373-1705>)
```

Comment:
Please use `terms:contributor` instead of `terms:creator`. I can make the change here.